Changes needed in server.cfg, mainly :
    Making use of RabbitMQ (default is ActiveMQ)
    The default port used to connect to RabbitMQ is 61613
    Default user : 'mcollective' & passwd : 'changeme'
    Change RabbitMQ to log to /dev/stderr, so it goes to the SMF log
    Set the PSK type explicitly to UID and the default to 'changeme'

--- marionette-collective-2.8.8/etc/server.cfg.dist.orig	2016-09-19 08:51:05.719526817 -0700
+++ marionette-collective-2.8.8/etc/server.cfg.dist	2016-09-19 08:51:49.506159302 -0700
@@ -1,20 +1,28 @@
+#######################################################################
+# Oracle has modified the originally distributed contents of this file.
+#######################################################################
+
 main_collective = mcollective
 collectives = mcollective
-libdir = /usr/libexec/mcollective
-logfile = /var/log/mcollective.log
+libdir = /usr/share/mcollective
+logger_type = file
+# Send output to SMF service log
+logfile = /dev/stderr
 loglevel = info
 daemonize = 1
 
 # Plugins
 securityprovider = psk
-plugin.psk = unset
+plugin.psk = changeme
+plugin.psk.callertype = uid
 
-connector = activemq
-plugin.activemq.pool.size = 1
-plugin.activemq.pool.1.host = stomp1
-plugin.activemq.pool.1.port = 6163
-plugin.activemq.pool.1.user = mcollective
-plugin.activemq.pool.1.password = marionette
+connector = rabbitmq
+plugin.rabbitmq.vhost = /mco
+plugin.rabbitmq.pool.size = 1
+plugin.rabbitmq.pool.1.host = localhost
+plugin.rabbitmq.pool.1.port = 61613
+plugin.rabbitmq.pool.1.user = mcollective
+plugin.rabbitmq.pool.1.password = changeme
 
 # Facts
 factsource = yaml
